kickstarter/android-oss
# NT-1648: Email Verification Interstitial UI

# 📲 What
Included in this PR: 

- Interstitial layout, viewmodel, and fragment
- Button added to internal tools screen to launch the interstitial fragment
- Added new copy 
- Added new email icon drawable

# 👀 See

 ![Screenshot_1604659528](https://user-images.githubusercontent.com/19390326/98358068-23eabe00-1ff4-11eb-8a50-c70f712ebc23.png) ![Screenshot_1604659516](https://user-images.githubusercontent.com/19390326/98358067-23eabe00-1ff4-11eb-93fd-42bc2ca74fdb.png)

# 📋 QA

- Tap on the overflow menu on the discovery screen
- Tap "internal tools" 
- Scroll to the bottom of this screen, should see a green button that reads "Email Verification Interstitial"
- Tap this button, should see the interstitial fragment popup.

# Story 📖

[Build Interstitial Screen UI](https://kickstarter.atlassian.net/browse/NT-1648)
